src/test/java/alinaTinca/tests/Tema5/Demoqa/Elements/BrokenLinksImagesClass.java

```diff
@@ -0,0 +1,38 @@
+package alinaTinca.tests.Tema5.Demoqa.Elements;
+
+import AlinaTinca.BrowserManagerClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BrokenLinksImagesClass {
+
+    public static void main(String[] args) {
+        goToTheBrokenLinksImages();
+        //check if the method called goToTheBrokenLinksImages is done successfully
+    }
+
+    public static void goToTheBrokenLinksImages(){
+        //create a method called goToTheBrokenLinksImages that goes to the Broken Links-Images section and identifies elements and make operations on them
+        ChromeDriver driver = BrowserManagerClass.createChromeDriver();
+        //create and configure the Chrome browse page settings
+        driver.get("https://demoqa.com/broken");
+        //get to the Demoqa URL
+        WebElement BrokenLinksImagesSection = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[7]/span"));
```

Comment:
When using xpath as a selector try to avoid using absolute xpath(meaning that you start from the html tag) and use relative xpath(meaning that you'll try to minimize the xpath by starting with the closest unique parent and continuing from there) instead. For example "/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[7]/span" would become "//div[@class='element-list collapse show']/ul/li[7]"

Reply:
class is not with capital letter there. it seems that there is a user with this username